docs/installation/configuration-file/_index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "Configuration File"
+linkTitle: "Configuration File"
+weight: 30
+no_list: true
+type: docs
+draft: false
+icon: "/installation/img/thumbnails/manage.png"
+images: ["/installation/img/thumbnails/manage.png"]
+description: "Building your robot configuration file for use with viam-server."
+---
+

```

Comment:
For people that find this link from the ToC (or otherwise out of context) I'd suggest an alert box at the top here staying that you don't need ANY of this if you use the normal cloud configuration system in App.

Reply:
Added copy to intro pointing to app instead, and alerts to both local-only options mentioning that neither is needed if app config was pursued instead.